### ISSUE TITLE
[CLI] fix description typos

### ISSUE THREAD

**Issue by daro1337** [scramjetorg/transform-hub#396]
**Describe the bug**
1.description typo for loglevel:
```
  apiUrl <apiUrl>      Set the hub API url
  logLevel <logLevel>  Set the hub API url
```

2. Some descriptions start with uppercase letters, some with lowercase

```
Options:
  -L, --log                   Logs all API requests in detail (default: true)
  -a, --api-url <url>         Specify base API url (default:
                              "http://141.95.92.157/api/v1/404//")
  -f, --format <value>        Specify display formatting: json or pretty (default:
                              "pretty")
  -h, --help                  display help for command

Commands:
  auth [command]
  scope|s                     Manage scope of the space and hub in temporary file.
  pack [options] <directory>
  host [command]              something
  hub                         allows to run programs in different data centers,
                              computers or devices in local network
  config|c                    configuration file operations
  scope|s                     Manage scope of the space and hub in temporary file.
  sequence|seq [command]      Performs operations on sequences
  space|spc                   operations on grouped and separated runtime
                              environments that allow sharing the data within them
  instance|inst [command]     operations on instance
  topic [command]             operations on topic
  completion                  completion operations
  help [command]              display help for command
```

**Steps to Reproduce**
1. `si config `
```
Usage: si config|c [options] [command]

configuration file operations

Options:
  -h, --help           display help for command

Commands:
  print|p              Print out the current config
  apiUrl <apiUrl>      Set the hub API url
  logLevel <logLevel>  Set the hub API url
  set <key> <value>    Set config value
  get <key>            Get config value
  unset|del <key>      Unset config value
  help [command]       display help for command
```

***Reproduction repo/branch***

**Expected behavior**
```
  apiUrl <apiUrl>      Set the hub API url
  logLevel <logLevel>  Set log level
```

**Additional information like logs, screenshots etc.**

**Comment by alicja-gruzdz** [comment]
Thanks @daro1337 . These should be corrected in https://github.com/scramjetorg/transform-hub/pull/364 which comes with many CLI improvments. I've checked the`logLevel` and it has a correct description there. I'll go through all descriptions later today.

**Comment by ErykSol** [comment]
Fixed in #364.
